include/ddc/kernels/fft.hpp

```diff
@@ -8,11 +8,11 @@
 #include <type_traits>
 #include <utility>
 
-#include <ddc/ddc.hpp>
-
 #include <KokkosFFT.hpp>
 #include <Kokkos_Core.hpp>
 
+#include "ddc/ddc.hpp"
```

Comment:
```suggestion
#include <ddc/ddc.hpp>
```

Here `ddc.hpp` is not in a `ddc` subdirectory of the current `kernels` directory.